frontend/src/component/feedbackNew/FeedbackComponent.tsx

```diff
@@ -206,6 +209,14 @@ export const FeedbackComponent = () => {
         setSelectedScore(event.target.value);
     };
 
+    const userType = isPro()
+        ? 'pro'
+        : isOss()
+          ? 'oss'
+          : isEnterprise()
+              ? 'enterprise'
+              : 'unknown';
```

Comment:
Would prefer a more readable way of defining this.

Reply:
Changed.

Reply:
Looks the same? Triple nested ternary takes up a lot of mental space when reading/scanning. Would prefer something more verbose but clear like: 

```
  const getUserType = () => {
        if (isPro()) {
            return 'pro';
        }

        if (isOss()) {
            return 'oss';
        }

        if (isEnterprise()) {
            return 'enterprise';
        }

        return 'unknown';
    };
```